status/flowcell.py

```diff
@@ -121,9 +121,12 @@ def get(self, flowcell_id):
                         modified_proj_name = 'undetermined'
                     else:
                         modified_proj_name = proj.replace('__','.')
-                    sum_project_lane_yield = sum(int(lane['clustersnb'].replace(',','')) for lane in lane_details if lane['Project']==proj)
-                    weighted_mean_q30 = sum(int(lane['clustersnb'].replace(',',''))*float(lane['overthirty']) for lane in lane_details if lane['Project']==proj and lane['overthirty'])/sum_project_lane_yield
-                    proj_lane_percentage_obtained = sum_project_lane_yield/total_lane_yield*100
+                    sum_project_lane_yield = sum(int(lane['clustersnb'].replace(',','')) for lane in lane_details if lane['Project']==proj and lane['clustersnb'])
+                    if sum_project_lane_yield:
+                        weighted_mean_q30 = sum(int(lane['clustersnb'].replace(',',''))*float(lane['overthirty']) for lane in lane_details if lane['Project']==proj and lane['clustersnb'] and lane['overthirty'])/sum_project_lane_yield
+                    else:
+                        weighted_mean_q30 = 0
+                    proj_lane_percentage_obtained = sum_project_lane_yield/total_lane_yield*100 if total_lane_yield else 0
```

Comment:
`sum_project_lane_yield/total_lane_yield*100` Using parentheses to clarify precedence would be nice